interface/forms/care_plan/templates/careplan_reason_row.php

```diff
@@ -45,11 +45,11 @@
                 </div>
                 <div class="col-md-6 form-group">
                     <label><?php echo xlt("Reason Recording Date"); ?></label>
-                    <input type='text' name='reasonDateLow[]' class="form-control reason_start_date datepicker" value='<?php echo attr($obj["reason_date_low"]); ?>' title='<?php echo xla('yyyy-mm-dd HH:MM Reason Recording Date'); ?>' />
+                    <input type='text' name='reasonDateLow[]' class="form-control reason_start_date datepicker" value='<?php echo attr($obj["reason_date_low"] ?? ''); ?>' title='<?php echo xla('yyyy-mm-dd HH:MM Reason Recording Date'); ?>' />
```

Comment:
I thought I had this always defined, but if its throwing errors that's a good fix.  I have very similar code in the observation and vitals form so I'll need to check those too if this was necessary.